views/root.tmpl

```diff
@@ -85,6 +85,9 @@
             };
         }
 
+        var subsidyToString = function(x, y = 1) {
```

Comment:
note that back in golang, you can use `nbSubsidy.Developer.ToCoin()` to get the usual floating point amount in DCR.  But this is fine.  No need to change it.